src/main/java/com/google/javascript/gents/RemoveGoogScopePass.java

```diff
@@ -46,8 +48,9 @@ public void visit(NodeTraversal t, Node n, Node parent) {
     }
 
     switch (callName) {
-      case "goog.provide":
-        // Register the goog.provided namespaces, so that we can remove any aliases.
+      case "goog.provide":  // FALL-THROUGH
```

Comment:
nit: we only do FALL-THROUGH comments on case branches that contain code, multiple cases in a row are just regular code.